deliver/lib/deliver/runner.rb

```diff
@@ -73,10 +73,10 @@ def prepare_app_icons(options = {})
       return unless options[:metadata_path]
 
       default_app_icon_path = Dir[File.join(options[:metadata_path], "app_icon.{png,jpg}")].first
-      options[:app_icon] ||= default_app_icon_path if File.exist?(default_app_icon_path)
+      options[:app_icon] ||= default_app_icon_path if default_app_icon_path && File.exist?(default_app_icon_path)
 
       default_watch_icon_path = Dir[File.join(options[:metadata_path], "watch_icon.{png,jpg}")].first
-      options[:app_icon] ||= default_watch_icon_path if File.exist?(default_watch_icon_path)
+      options[:apple_watch_app_icon] ||= default_watch_icon_path if default_watch_icon_path && File.exist?(default_watch_icon_path)
```

Comment:
Whops, should that stay `options[: app_icon]` instead? Did you want to change this too?